src/lashup_config.erl

```diff
@@ -83,9 +84,14 @@ join_timeout() ->
   get_env(join_timeout, 250).
 
 %%
--spec(contact_nodes() -> [node()]).
+-spec(contact_nodes() -> ordsets:ordset(node())).
 contact_nodes() ->
-  get_env(contact_nodes, []).
+  Nodes = ordsets:from_list(get_env(contact_nodes, [])),
+  ordsets:del_element(node(), Nodes).
+
+-spec(contact_nodes([node()]) -> ok).
+contact_nodes(Nodes) ->
+  application:set_env(lashup, contact_nodes, Nodes).
```

Comment:
This is a change in behavior. Previously, the code was taking a union of `contact_nodes` + `extra_masters`. In new code the `extra_masters` is overriding the `contact_nodes`. Maybe we could take a union as we were doing before?

Reply:
We set `contact_nodes` only in tests. It actually doesn't change anything, it doesn't make any sense to support two independent sets of bootstrapping nodes in lashup.

Reply:
Actually, it does change the behavior. Imagine a customer scenario where both `contact_nodes` and `mesos_dns_resolvers` are set. Previous code would have returned a union, however, this code would just return `mesos_dns_resolvers`. Now, the question is would this change in behavior is acceptable. I guess the answer is `yes` as I doubt that any customer would have set the `contact_nodes` explicitly. Especially, when it is not exposed via config.yaml